docs/core/additional-tools/uninstall-tool-cli-dry-run.md

```diff
@@ -0,0 +1,189 @@
+    [-v|--verbosity <LEVEL>] [--force]
+
+dotnet-core-uninstall dry-run <TARGET> [--x64|--x86] <FILTER>
+    [-v|--verbosity <LEVEL>]
+
+dotnet-core-uninstall dry-run -h|--help|-?
+```
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+```dotnetcli
+dotnet-core-uninstall dry-run <TARGET> <VERSION>...
+    [-v|--verbosity <LEVEL>] [--force] [-y|--yes]
+
+dotnet-core-uninstall dry-run <TARGET> <FILTER>
+    [-v|--verbosity <LEVEL>] [--force] [-y|--yes]
+
+dotnet-core-uninstall dry-run -h|--help|-?
+```
+
+::: zone-end
+
+## Description
+
+The `dotnet-core-uninstall list` command simulates .NET SDK and runtime removal. A status output is provided for each .NET SDK and runtime that would have been removed by the tool.
+
+### Arguments
+
+**`TARGET`**
+
+  The specified type you want to uninstall. Valid options are listed in the [Options - TARGET](#options---target) section.
+
+**`VERSION`**
+
+  The specified version to uninstall. You can list several versions separated by a space. Response files are also supported.
+
+  > [!TIP]
+  > Response files are an alternative to placing all the versions on the command line. They're text files, typically with a *\*.rsp* extension, and each version is listed on a separate line. To specify a response file for the `VERSION` argument, use the \@ character immediately followed by the response file name.
+
+**`FILTER`**
+
+  Specifies a value used to filter the `TARGET`. Valid options are listed in the [Options - FILTER](#options---filter) section.
+
+## Options - TARGET
+
+::: zone pivot="os-windows"
+
+- **`--aspnet-runtime`**
+  
+  Discovers all the ASP.NET Core runtimes that can be uninstalled with this tool.
+
+- **`--hosting-bundle`**
+  
+  Lists all the .NET hosting bundles that can be uninstalled with this tool.
+
+::: zone-end
+
+- **`--runtime`**
+
+  Lists all the .NET runtimes that can be uninstalled with this tool.
+
+- **`--sdk`**
+
+  Lists all the .NET SDKs that can be uninstalled with this tool.
+
+::: zone pivot="os-windows"
+
+- **`--x64`**
+
+  Lists all the x64 .NET SDKs and runtimes that can be uninstalled with this tool.
+
+  > [!NOTE]
+  > If `--x64` or `--x86` isn't specified, then both x64 and x86 will be removed.
+
+- **`--x86`**
+
+  Lists all the x86 .NET SDKs and runtimes that can be uninstalled with this tool.
+
+  > [!NOTE]
+  > If `--x64` or `--x86` isn't specified, then both x64 and x86 will be removed.
+
+::: zone-end
+
+## Options - FILTER
+
+These options are exclusive.
+
+- **`--all`**
+
+  Removes all the .NET SDKs and runtimes.
+
+- **`--all-below <VERSION>[ <VERSION>...]`**
+
+  Removes only the .NET SDKs and runtimes with a version smaller than the specified version. The specified version remains installed.
+
+- **`--all-but <VERSION>[ <VERSION>...]`**
+
+  Removes all the .NET SDKs and runtimes, except those versions specified.
+
+- **`--all-but-latest`**
+
+  Removes the .NET SDKs and runtimes, except the highest version.
```

Comment:
```suggestion
  Removes the .NET SDKs and runtimes, except the latest version.
```

Reply:
All this wording came from the original PM that wrote these docs. I'm not sure if there's some sort of nuance between highest and latest. Like is latest interpreted as latest version (.NET 8 vs .NET 7) or is highest interpreted between versions (.NET 8.0.100 vs .NET 8.0200 vs .NET 7.0.201, would keep .NET 8.0200 vs .NET 7.0.201 as the highest versions of major releases)

Reply:
@sfoslund Can you comment?

Reply:
I guess it's @marcpopMSFT now!